src/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ source_group("Source Files" FILES ${imCommon_SOURCES})
 add_executable(imaginaryMyst WIN32 ${imCommon_HEADERS} ${imCommon_SOURCES})
 target_link_libraries(imaginaryMyst imAnim imScene imSound imSurface imVfs)
 target_link_libraries(imaginaryMyst squish)
-target_link_libraries(imaginaryMyst ${SDL_LIBRARY} ${OPENGL_LIBRARIES})
+target_link_libraries(imaginaryMyst ${SDL2_LIBRARIES} ${OPENGL_LIBRARIES})
```

Comment:
This didn't work for me -- I had to use the CMake import target (`SDL2::SDL2`) to get it to link.